doc/guides/west/build-flash-debug.rst

```diff
@@ -61,6 +85,10 @@ no additional parameters.
   whether ``-b`` is required, just try leaving it out. West will print an
   error if the option is required and was not given.
 
+.. tip::
+  You can use the :ref:```west boards`` <west-boards>` command to list all
```

Comment:
Nice try.  That's not going to work. (Check the [doc build artifacts for this PR](https://builds.zephyrproject.org/zephyrproject-rtos/zephyr/15704/docs/guides/west/build-flash-debug.html#building-west-build))
```suggestion
  You can use the :ref:`west boards <west-boards>` command to list all
```

Reply:
Fixed